doc/man/8/rbd.rst

```diff
@@ -668,20 +666,22 @@ Commands
 :command:`trash ls` [*pool-name*]
   List all entries from trash.
 
-:command:`trash mv` *image-spec*
+:command:`trash mv` [--expires-at <expires-at>] *image-spec*
   Move an image to the trash. Images, even ones actively in-use by 
-  clones, can be moved to the trash and deleted at a later time.
+  clones, can be moved to the trash and deleted at a later time. Use
+  --expires-at to set the expiration time of an image after which
```

Comment:
`--expires-at`

Reply:
Sorry, did not understand this comment.

Reply:
I think Anthony meant "add backticks around --expires-at" here.

Reply:
Done.